denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc

```diff
@@ -0,0 +1,45 @@
+armor_degradation:
+  type: world
+  debug: false
+  events:
+    on player damaged by slime:
+      - if <context.damager.is_mythicmob> && <context.damager.MythicMob.level.is[MORE].to[5]>:
+          - determine passively cancelled
+          - choose <util.random.int[1].to[4]>:
+              - case 1
+                - if <player.equipment_map.get[helmet].material.name> == *_helmet|turtle_helmet
+                  - define slot 40
+                  - define item <player.equipment_map.get[helmet].material.name>
+                  - hurt <context.damager.mythicmob.level.multiply[3]>
+                - else
+                  - hurt <context.damager.mythicmob.level.multiply[5]>
+                  - narrate "<&e>Acid splashes on you, burning your face."
+              - case 2
+                - if <player.equipment_map.get[chestplate].material.name> == *_chestplate
+                  - define slot 39
+                  - define item <player.equipment_map.get[helmet].material.name>
+                  - hurt <context.damager.mythicmob.level.multiply[3]>
+                - else
+                  - hurt <context.damager.mythicmob.level.multiply[5]>
```

Comment:
`Multiply` is not a valid sub-tag, you likely meant `ElementTag.mul[#.#]`.
missing colon on `else` statement.
```suggestion
                  - hurt <context.damager.mythicmob.level.mul[3]>
                - else:
                  - hurt <context.damager.mythicmob.level.mul[5]>
```